puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/nightly_packaging.groovy

```diff
@@ -0,0 +1,30 @@
+def setup_nightly_build_environment(args) {
+    def commit_hash = ''
+
+    dir(args.package_name) {
+        git(url: "https://github.com/${args.github_repo}.git", branch: 'develop')
```

Comment:
make the branch configurable, as some projects use `master` instead of `develop`

Reply:
Yeah, that's a good point. I was going to add this feature down the line but might as well do it now.